src/PageMetrics.cpp

```diff
@@ -1,106 +1,115 @@
 #include "PageMetrics.h"
 
 #include <QApplication>
-#include <QDesktopWidget>
+#include <QScreen>
+#include <QMainWindow> // Let's get current screen here
```

Comment:
Maybe a better solution will be to use `QGuiApplication::primaryScreen` here?

Reply:
Maybe, but docs says that `QGuiApplication::primaryScreen` returns the screens were QWindow are created, while this hack will get current screen os QMainWindow

Reply:
What worried me, that if the app has multiple windows, then metrics will depend on the main window and if it will be moved from one screen to another, metrics for the secondary windows (which are stayed in place) will be changed.